src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/Win32ThreadPoolBoundHandle.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Threading
     //
```

Comment:
This should be moved under libraries

Reply:
(It is fine to depend on the `UseWindowsThreadPool` condition to be optimized and the code that is under the false branch to be dead-code eliminated.)